multinet/multinet/multinet.py

```diff
@@ -30,6 +30,9 @@ def graphql_query(query, variables=None):
             if excess > 0:
                 app.logger.error(f'{excess} more error{"s" if excess > 1 else ""}')
 
+            raise ValueError("Error when connecting to the DB.")
+            return
```

Comment:
This `return` seems to not be necessary, owing the the raised exception in the line above.

Reply:
Removed the return.